### ISSUE TITLE
Screen reader accessibility bug in the "issues" web interface of github: screen reader users cannot attach files to their issues

### ISSUE THREAD

**Issue by Palacee-hun** [isaacs/github#1898]
Github web interface for the most part plays very nicely with screen readers. By the way screen readers are essential software for the blind and visually impaired to operate computers. Just look up "screen reader" on wikipedia and the introductory section tells the most important facts about them.

I have found only one major problem with Github web interface so far: I cannot attach any files to my issues.

I cite from Github docs:
``
To attach a file to an issue or pull request conversation, drag and drop it into the comment box. Alternatively, you can click the bar at the bottom of the comment box to browse, select, and add a file from your computer.
``
Drag and drop is not feasible at all for screen reader users, as it is a mouse-only operation, and screen reader users utilise keyboard-only workflows almost exclusively. And that bar at the bottom of the comment box is some non-standard graphical web element, which screen readers cannot gather info about at all and thus it is impossible to locate it with a screen reader. So screen reader users are caught in this respect.

I try to give some steps to reproduce:
1. Download the popular, free and open source Windows-only screen reader NVDA from www.nvaccess.org. By the way it is hosted on Github at https://github.com/nvaccess/nvda.
2. Run the downloaded installer. It plays some music and then comes up talking. There's no need to install it actually, there is an option to just keep it running and that's sufficient for this.
3. Right click the NVDA tray icon, choose Preferences submenu and then Preferences again. This brings up a dialog where most NVDA preferences can be set.
4. Click the visual effects tab and check the fourth option (visually highlight browse mode caret) and apply this setting by clicking OK. This will enable you to actually see visually the web element that NVDA speaks when navigating on a website.
5. Start opening any new issue on Github. When you are presented with the web interface to compose your issue, begin navigating around the website the NVDA way. Press f to jump to the next web control that NVDA identifies and shift-f to go to the previous one. NVDA speaks out the control it lands on and you can follow along visually. Note that it won't land on the bar for attaching files to the issue.
6. Now try navigating around with the keys k and shift-k (jump to next / previous link respectively). This will be unsuccessful too, you won't land on the bar this way, as it is not a link.

By these steps you can ascertain to a sufficient degree that the bar for attaching files is invisible to NVDA, and that the users of this screen reader won't be able to attach files. There are other screen readers, but they have totally analogous methods to navigate websites and the same results are to be expected.
The fix for this is to change the bar for attaching files to some standard HTML element, preferably a button with the label "Attach files". The same steps can be used to check if the fix actually works.

I use Github from Windows machines with the latest Chromium-based MS Edge, currently 88.0.705.50.
